### ISSUE TITLE
Error while g

### ISSUE THREAD

**Issue by FilipJurkovic** [creativetimofficial/material-tailwind#468]

**Comment by kelihansen** [comment]
I just encountered a very similar issue. In my case, I upgraded to Next 13.5, and I'm getting `Maximum call stack size exceeded` when I try to build the project for production with the material-tailwind Dialog included.

**Comment by Deeds67** [comment]
I have the same issue. I managed to work around it by using an older version of `next`

`"next": "13.5.4",`

Anything newer than that gives me the same `Maximum call stack size exceeded` error

**Comment by kelihansen** [comment]
> I have the same issue. I managed to work around it by using an older version of `next`
> 
> `"next": "13.5.3",`
> 
> Anything newer than that gives me the same `Maximum call stack size exceeded` error

Thank you, 13.5.3 is working for me, as well

**Comment by alexews** [comment]
I've faced same issue

**Comment by adrileo3** [comment]
I've faced this issue with Button component

**Comment by schekutiev** [comment]
I have the same problem since NextJs v 3.5.4. Downgrade is not the solution in my case, so I decided to drop material-tailwind...

**Comment by JohnMwendwa** [comment]
> I've faced this issue with Button component

I had the same issue and downgraded to `next@13.4.9` and don't run `npm audit fix` or `npm audit fix --force` after downgrading. This is just a temporary solution as I wait for `@material-tailwind/react` to be updated . You can check my code and the live  production link at [github-code](https://github.com/JohnMwendwa/vet-management-system) & [live-production-demo](https://vet-management-system.vercel.app/)

**Comment by kelihansen** [comment]
I've been following https://github.com/vercel/next.js/issues/56614 - does the latest Next.js canary fix the issue for anyone? It does not for me.

**Comment by NPJigaK** [comment]
When using Material Tailwind, downgrading the next version to 13.5.3 solves the problem. 13.5.4 or higher will cause an error. v13.5.7-canary.9 as well.

**Comment by kdy1** [comment]
Can you assign this issue to me?

**Comment by alexewsv** [comment]
I've still faced this error with material-tailwind@2.1.4 + nextjs@13.5.6. but downgrade nextjs to 13.4.19 work well. 
More info:  `npm run dev` is work well, but `npm run build` will raise error 

@kdy1 I see the issue in nextjs repo is closed but, it seem not. btw thank your hard word.

**Comment by kdy1** [comment]
@alexews `v13.5.7-canary.16` will work.

**Comment by alexewsv** [comment]
> @alexews `v13.5.7-canary.16` will work.

Yeah, great, confirmed it works with v13.5.7-canary.16 and also latest v13.5.7-canary.25 without this regression ;)

**Comment by aperepel** [comment]
Thank you for capturing these details, folks. I ran into this, downgrading didn't help much, but `v13.5.7-canary.25` works.

**Comment by manojadams** [comment]
I am facing this issue with following versions:
- "next": "14.0.3"
- "tailwindcss": "^3.3.0"
- "postcss": "^8"

**Comment by eri-prado** [comment]
I'm facing this issue with the version: "next": "13.5.6"

Problem resolved by upgrading to version: "next": "14.1.0"

🇧🇷🇧🇷
